plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -264,7 +315,9 @@ global proc mayaUsdMenu_createMenuCallback() {
 // mayaUsd_updateSelectionKindMenuItem
 // Update the menuItem radio button in the Selection Mode menu
 global proc mayaUsd_updateSelectionKindMenuItem() {
-    string $selectionKind = mayaUSD_getSelectionKind();
+    // Note: we do all string compares as case-insenstive (lowercase)
+    //       as the menu labels are capitalized.
+    string $selectionKind = `tolower(mayaUSD_getSelectionKind())`;
```

Comment:
Since Natalia asked for these menu items to be capitalized the string will no longer match the option var value. So do the string comparison in all lowercase.